sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,8 +1,33 @@
 # Release History
 
-## 5.1.0-preview.2 (Unreleased)
+## 5.1.0
 
-Changes will be detailed prior to release
+### Acknowledgments
+
+Thank you to our developer community members who helped to make the Event Hubs client libraries better with their contributions to this release:
+
+- Alberto De Natale _([GitHub](https://github.com/albertodenatale))_
```

Comment:
Yay!

Reply:
I feel like we should add that in ours since we've had a couple of contributions. :)

Reply:
I wanted to at least give the folks volunteering their time a small bit of recognition, since we haven't yet figured out if we're able to offer any more tangible thank you gifts.